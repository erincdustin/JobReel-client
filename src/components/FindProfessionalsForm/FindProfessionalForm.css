.ProfessionalSearchForm {
  width: 320px;
  border: 1px black solid;
  min-width: 300px;
  margin: 0 auto;
  padding: 5%;
  line-height: 150%;
  border-radius: 10px;
  text-align: center;
  margin: auto;
  background: #3b3a50;
  color: #eeebd9;
  box-shadow:  0.5em 0.5em 0.2em rgb(48, 39, 39);
}

.option {
  color: black;
}

.findProfessionalsBackgroundImage {
  position: relative;
  grid-column-start: 1;
  grid-column-end: 9;
}

.findProfessionalsBackgroundImage > img {
  height: 250px;
  width: 250px;
  position: absolute;
  top: -100px;
  left: 5%;
  z-index: -1;
}