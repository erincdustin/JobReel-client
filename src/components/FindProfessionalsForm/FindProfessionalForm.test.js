import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { BrowserRouter } from 'react-router-dom'
import FindProfessionalForm from './FindProfessionalsForm'

describe('FindProfessionalForm smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><FindProfessionalForm /></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('FindProfessionalForm snapshot test', () => {
  it('renders the UI as expected', () =>{
    const wrapper = shallow((<FindProfessionalForm />))
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})