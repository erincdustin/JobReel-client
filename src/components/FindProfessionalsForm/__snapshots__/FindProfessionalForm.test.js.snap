// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FindProfessionalForm snapshot test renders the UI as expected 1`] = `
<div
  className="job-search"
>
  <SideNav />
  <form
    className="JobSearchForm"
    onSubmit={[Function]}
  >
    <h4>
      Enter a domain or company name
    </h4>
    <div>
      <div>
        <Label
          htmlFor="domain-input"
        >
          Domain
        </Label>
        <br />
        <ForwardRef
          disabled={null}
          id="domain-input"
          name="domain"
          onChange={[Function]}
          required={true}
        />
      </div>
      <div>
        <Label
          htmlFor="company-input"
        >
          Company
        </Label>
        <br />
        <ForwardRef
          disabled={null}
          id="company-input"
          name="company"
          onChange={[Function]}
          required={true}
        />
      </div>
    </div>
    <div>
      <Label
        htmlFor="seniority-input"
      >
        Seniority
      </Label>
      <br />
      <StateManager
        className="basic-multi-select"
        classNamePrefix="select"
        defaultInputValue=""
        defaultMenuIsOpen={false}
        defaultValue={null}
        id="seniority-input"
        isMulti={true}
        makeAnimated={true}
        name="seniority"
        options={
          Array [
            Object {
              "label": "Junior",
              "value": "junior",
            },
            Object {
              "label": "Senior",
              "value": "senior",
            },
            Object {
              "label": "Executive",
              "value": "executive",
            },
          ]
        }
        styles={
          Object {
            "option": [Function],
            "singleValue": [Function],
          }
        }
      />
    </div>
    <div>
      <Label
        htmlFor="department-input"
      >
        Department
      </Label>
      <br />
      <StateManager
        className="basic-multi-select"
        classNamePrefix="select"
        defaultInputValue=""
        defaultMenuIsOpen={false}
        defaultValue={null}
        id="department-input"
        isMulti={true}
        makeAnimated={true}
        name="department"
        options={
          Array [
            Object {
              "label": "IT",
              "value": "it",
            },
            Object {
              "label": "Finance",
              "value": "finance",
            },
            Object {
              "label": "Management",
              "value": "management",
            },
            Object {
              "label": "Sales",
              "value": "sales",
            },
            Object {
              "label": "Legal",
              "value": "legal",
            },
            Object {
              "label": "Support",
              "value": "support",
            },
            Object {
              "label": "HR",
              "value": "hr",
            },
            Object {
              "label": "Marketing",
              "value": "marketing",
            },
            Object {
              "label": "Communication",
              "value": "communication",
            },
            Object {
              "label": "Executive",
              "value": "executive",
            },
          ]
        }
        styles={
          Object {
            "option": [Function],
            "singleValue": [Function],
          }
        }
      />
    </div>
    <br />
    <ForwardRef
      type="submit"
    >
      Submit
    </ForwardRef>
  </form>
</div>
`;
